src/odb/src/lefout/lefout.cpp

```diff
@@ -282,7 +328,7 @@ void lefout::writeBlock(dbBlock* db_block)
   fprintf(_out, "  ORIGIN %g %g ;\n", origin_x, origin_y);
   fprintf(_out, "  SIZE %g BY %g ;\n", size_x, size_y);
   writePins(db_block);
-  writeObstructions(db_block);
+  writeObstructions(db_block, true);
```

Comment:
Should this be a user option rather than true?

Reply:
Do we want the option to enable, or disable clearance? My first attempt looks like:
```
+sta::define_cmd_args "write_abstract_lef" {[-pin_clearance] filename}
```